hack/bicep-types-radius/src/autorest.bicep/test/integration/generated/basic/test.rp1/2021-10-31/types.md

```diff
@@ -37,20 +26,13 @@
 * **lastModifiedBy**: string: The identity that last modified the resource.
 * **lastModifiedByType**: 'Application' | 'Key' | 'ManagedIdentity' | 'User': The type of identity that created the resource.
 
+## TestType1Properties
+### Properties
+* **basicString**: string: Description for a basic string property.
+* **stringEnum**: 'Bar' | 'Foo': Description for a basic enum property.
+
 ## TrackedResourceTags
 ### Properties
 ### Additional Properties
 * **Additional Properties Type**: string
```

Comment:
FoosRequest. FoosResponse still exists under :
hack/bicep-types-radius/src/autorest.bicep/test/integration/generated/basic/test.rp1/2021-10-31/types.md and
sk593/radius/hack/bicep-types-radius/src/autorest.bicep/test/integration/generated/basic/test.rp1/2021-10-31/types.json .. should it be deleted here?

Reply:
This might be because of how the `bicep-types` markdown generator is written. Ideally, it should still be generated but I'll double check if this was missed in that repo or if it's an issue in our code

Reply:
Following up here: We need some clarity/more info on the .md files under the hack/bicep-types-radius/generated/ folder not added to github (they are in .gitignore folder) but then there is a test folder which also includes these generated files which gets into github and there is some discrepancy in there.

Reply:
I think the test folder can keep the MD files in case we ever add them back to the generated types folder. From what I remember, the other MD files were added to the .gitignore because there was a discrepancy between the MD files and the docs page and we wanted to keep them the same. @willtsai or the PMs may have more insight on this. This is the PR that removed the MD files from the hack folder: https://github.com/radius-project/radius/pull/6331

Reply:
okay, so it looks like Aaron removed the MD files from the hack folder in [PR#6331](https://github.com/radius-project/radius/pull/6331) because these MD documents gets autogenerated in the workflow added by the same PR. Thus, I believe the right call is to continue keeping the MD files in the .gitignore so that the docs site can depend on the autogeneration workflow that keeps the documentation (for [example](https://docs.radapp.io/reference/api/applications.core/applications/)) up to date.

Reply:
sounds good, thanks for looking into that. I still think it makes sense to keep the MD file in the test folder since we do generate those (even if they're not uploaded to Github). since the types in that file are just test resources, I don't think it'll be a point for confusion. thoughts @lakshmimsft ?